media/stylus/components/structure.styl

```diff
@@ -54,6 +54,16 @@ draw-grid();
 
 #tabzilla {
     bidi-value(float, right, left);
+    display: block;
+    height: 44px;
+    width: 150px;
+    overflow: hidden;
+}
+
+#tabzilla-panel {
+    /* in case the javascript loads before the CSS */
+    height: 0px;
+    overflow: hidden;
```

Comment:
Awesome! :smile:

Is this block of additions copied from Tabzilla assets? If so, some comments to that effect might help us to keep the properties up-to-date over time.